extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -150,11 +151,11 @@ private function getGlobalBotIds() {
 			WikiaResponse::CACHE_LONG,
 			function () {
 				global $wgExternalSharedDB;
-				$db = wfGetDB( DB_SLAVE, [], $wgExternalSharedDB );
+				$db = wfGetDB( DB_SLAVE, [ ], $wgExternalSharedDB );
 
 				$sqlData = ( new WikiaSQL() )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$sqlData". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)